### ISSUE TITLE
1.6.1.11: Plot limits not adjusted!

### ISSUE THREAD

**Issue by CurePindal** [MyEssentials/MyTown#25]
So, this is a bit weird.

Lets assume I use the node "mytown.mayor.blocks.10" and found a town it starts with their 10 blocks. When I change that blocklimit to "mytown.mayor.blocks.20" it has no impact on the existing town - not even after a server restart.
Same issue when you change the node via a PEX group with a different limit - it has no impact, which makes me a sad panda.

Guess its not intentional?

**Comment by jgoett154** [comment]
I need to rewrite the way plot limit works because the perm nodes just don't make much sense. As far as it not updating, it should and have no idea why its not. I am investigating as we speak. Since I wasn't the one who initially wrote MyTown, I just picked it up after the old author, it may take me some time to find where the issue lies. I have been slightly tempted to rewrite MyTown almost completely because there are things I would change greatly, but I have avoided doing so because it would mean a very long wait for server owners. Anyways, I will find out the source of the issue ASAP.
